admin/src/components/Collections.js

```diff
@@ -0,0 +1,111 @@
+/**
+ *
+ * Block
+ */
+
+import React, { memo, useState, useEffect } from 'react'
+import { request } from 'strapi-helper-plugin'
+import pluginId from '../pluginId'
+import { Table } from '@buffetjs/core'
+import { errorNotifications, successNotification } from '../utils/notifications'
+import { Wrapper } from '../components/Wrapper'
```

Comment:
Tu peux tester de faire des absolute imports. Dans un fichier `jsconfig.json` à la racine essaie de rajouter ça : 
```
{
  "compilerOptions": {
    "baseUrl": "admin/src"
  }
}
```

Ensuite, tu pourras import directement avec `import { Wrapper } from 'components/Wrapper'

Reply:
I couln't find how to make it work. I wonder if it is related to the fact that the plugin will be compiled following the compiler rules of the strapi application that uses it. 

I have looked at multiple other plugins and they used the `...` as well. I will open an issue about it but thanks a lot for the information!